packages/near-fast-auth-signer/src/components/CreateAccount/CreateAccount.tsx

```diff
@@ -119,6 +120,16 @@ function CreateAccount() {
   const [searchParams] = useSearchParams();
   const [inFlight, setInFlight] = useState(false);
 
+  const contractId = searchParams.get('contract_id');
+  useEffect(() => {
```

Comment:
This is duplicated, isn't better create a custom hook for it? 

Or call it only from the App.ts? (you set it to run only for routes that aren't `/multi-chain`)